src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -700,6 +708,10 @@ applyExec initialGas interp em senderSigs hsh nsp = do
     for_ _erLogGas $ \gl -> gasLog $ "gas logs: " <> sshow gl
     logs <- use txLogs
     rk <- view txRequestKey
+
+    -- set tx warnings to eval warnings
+    txWarnings .= _erWarnings
```

Comment:
`txWarnings` will never have an existing value here?

Reply:
Perhaps then you could just add an assert that txWarnings is always empty here? It would be helpful to know that this is an assumption, and that nothing will be thrown away. Alternatively, we could use `<>=`.